App/TopTrumps/TopTrumps.Data/Services/IUsersService.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace TopTrumps.Data.Services
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using DTOs;
+
+    public interface IUsersService
+    {
+        Task<User> GetUser(int id);
+
+        Task<IEnumerable<User>> GetUsers();
+
+        Task<User> GetAllUsers();
```

Comment:
What is the intended difference between `GetUsers` and `GetAllUsers` ? Sounds like the should do the same thing? Also the return type for `GetAllUsers` will only return a Task that returns a single user. The method name sounds like it would return multiple users.